lib/cob_index/indexer_config.rb

```diff
@@ -88,9 +91,17 @@
 
 # Creator/contributor fields
 to_field "creator_t", extract_marc("245c:100abcdejlmnopqrtu:110abcdelmnopt:111acdejlnopt:700abcdejqu:710abcde:711acdej", trim_punctuation: true), delete_if(CORPORATE_NAMES), wrap_begin_end
+to_field "creator_authority_record_id_ms", extract_marc("1000:1100:1110")
+to_field "creator_real_world_object_uri_ms", extract_marc("1001:1101:1111")
```

Comment:
This field name feels a bit awkward / long. @htomren -- do you have any suggestions?

Reply:
To clairfy, I'm referring to "creator_real_world_object_uri_ms" and other examples of "real_world_object" in field name. @dkinzer , I'm not sure exactly what this means and why we're not just saying "creator_uri_ms"?

Reply:
Ok, sounds good! I wasn't familar with that terminology.